src/functionalTest/java/uk/gov/hmcts/probate/functional/checkyouranswers/SolCcdServiceCheckYourAnswersTests.java

```diff
@@ -70,8 +70,7 @@ public void verifyLegalStatementAcceptInTheReturnedPDF() {
 
     @Test
     public void verifyLegalStatementSolicitorsDeclarationInTheReturnedPDF() {
-        validatePostRequestSuccessForLegalStatement("The executors believe that all the information stated in the legal statement is true. They have authorised \n" +
-                "SolicitorFirmName to sign a statement of truth on their behalf.");
+        validatePostRequestSuccessForLegalStatement("The executors believe that all the information stated in the legal statement is true.");
```

Comment:
you might want to consider to add new version of next sentence after - 

The executors believe that all the information stated in the legal statement is true.